README.md

```diff
@@ -0,0 +1,196 @@
+# PyFlexPlot
+PyFlexPlot is a Python-based tool to visualize FLEXPART dispersion simulation results stored in NetCDF format.
+## Table of Contents
+- [Prerequisites and Cloning the Repository](#prerequisites-and-cloning-the-repository)
+- [Getting Started](#getting-started)
+- [Usage](#usage)
+  - [Usage Example](#usage-example)
+- [Developer Notes](#developer-notes)
+  - [Implemented Debugging Features](#implemented-debugging-features)
+  - [Roadmap to your first Contribution](#roadmap-to-your-first-contribution)
+  - [Testing and Coding Standards](#testing-and-coding-standards)
+    - [Pre-commit on GitHub Actions](#pre-commit-on-github-actions)
+    - [Jenkins](#jenkins)
+- [Features](#features)
+- [Credits](#credits)
+- [External Links](#external-links)
+- [License](#license)
+## Prerequisites and Cloning the Repository
+Before you get started with this repository, ensure you have the following software/tools installed on your system: Git, Python and conda/mamba.
+To get a local copy of this repository, run following commands and naviate into the repository:
+```bash
+git clone https://github.com/MeteoSwiss-APN/pyflexplot <custom_directory_name>
+cd <custom_directory_name>
+```
+
+## Getting Started
```

Comment:
This "getting started" focuses on development. Maybe it would be helpfull to have a quickstart section for people that just want to use it? So pinned dependencies, non-editable installation,...

Reply:
Yes perfect!